be/src/runtime/exec_env.cpp

```diff
@@ -373,6 +373,17 @@ Status ExecEnv::_init_storage_page_cache() {
         LOG(WARNING) << "Config storage_page_cache_limit is greater than memory size, config="
                      << config::storage_page_cache_limit << ", memory=" << MemInfo::physical_mem();
     }
+    // If pagecache is enabled and the value is not set or is set to 0, the max default value is
+    // the smaller of either:
+    // - kcacheSizeInitRatio * system memory size
+    // - kcacheSizeInit
+    if (!config::disable_storage_page_cache && storage_cache_limit == 0) {
```

Comment:
```
if (!config::disable_storage_page_cache) {
      if (storage_cache_limit == 0) {
      } else {
      }
}
```

Reply:
if `storage_cache_limit` less then one size, such as 1G, disable the page cache, otherwise, there may be a lot of swaps in and out